File: src/main/java/commands/listeners/OnStaticReactionRemoveListener.java
package commands.listeners;

import net.dv8tion.jda.api.entities.Message;
import net.dv8tion.jda.api.events.message.guild.react.GuildMessageReactionRemoveEvent;

public interface OnStaticReactionRemoveListener {

    void onStaticReactionRemove(Message message, GuildMessageReactionRemoveEvent event) throws Throwable;

    String titleStartIndicator();

}

File: src/main/java/commands/listeners/OnStaticReactionRemoveListener.kt
package commands.listeners

import net.dv8tion.jda.api.entities.Message
import net.dv8tion.jda.api.events.message.react.MessageReactionRemoveEvent

interface OnStaticReactionRemoveListener {

    @Throws(Throwable::class)
    fun onStaticReactionRemove(message: Message, event: MessageReactionRemoveEvent)

}